src/mixins/java/org/spongepowered/common/mixin/api/minecraft/world/level/LevelMixin_API.java

```diff
@@ -483,6 +484,13 @@ public boolean setBiome(final int x, final int y, final int z, final Biome biome
         return entityList;
     }
 
+    @Override
+    public Collection<? extends Entity> entities() {
+        ImmutableList.Builder<Entity> list = ImmutableList.builder();
+        this.loadedChunks().forEach(chunk -> list.addAll(chunk.entities()));
```

Comment:
is `loadedChunks()` an API method? I almost don't want to use it explicitly because I don't know what the semantics are of loadedChunks() by itself.

Reply:
I've removed this implementation in LevelMixin, instead it has implementations in Server and Client seperately, the server implementation already exists.